### PR TITLE
feat: add new resources to existing file

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

- adds option to save new resource to existing file

## Fixes

- Change style of load/reload action button

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/NewResourceWizard/NewResourceWizard.tsx [hunk 1]
```diff
@@ -334,21 +365,29 @@ const NewResourceWizard = () => {
...
-        {fileMap[ROOT_FILE_ENTRY] ? (
-          <SaveToFolderWrapper>
-            <StyledCheckbox onChange={e => setShouldSaveState(e.target.checked)} checked={shouldSaveToFolder}>
-              Save to folder
-            </StyledCheckbox>
+
+        <SaveDestinationWrapper compact>
+          <StyledSelect value={savingDestination} onChange={handleSavingDestinationChange}>
+            <Option value="saveToFolder">Save to folder</Option>
+            <Option value="saveToFile">Save to file</Option>
```
Comment (olensmar) [inline src/components/organisms/NewResourceWizard/NewResourceWizard.tsx:372]: let's call this "Add to existing file"

**Comment by olensmar** [comment]
need to fix sizing - 
image

File: src/components/organisms/FileTreePane/FileTreePane.tsx [hunk 2]
```diff
@@ -425,6 +414,17 @@ const TreeItem: React.FC<TreeItemProps> = props => {
         onClick={e => {
           e.domEvent.stopPropagation();
 
+          onCreateResource(isFolder ? {targetFolder: target} : {targetFile: target});
+        }}
+        key="create_resource"
+      >
+        New Resource
```
Comment (olensmar) [inline src/components/organisms/FileTreePane/FileTreePane.tsx:421]: Can we call it "Add Resource" for files and "New Resource" for folders?

Comment (erdkse) [reply src/components/organisms/FileTreePane/FileTreePane.tsx:421]: yes

**Comment by olensmar** [comment]
thanks @erdi - see below - could we make the left dropdown smaller so more of the filename is shown? perhaps we should call the option "Add to file"  to make it even smaller?

image

**Review by olensmar — APPROVED** [review]
great - thank you!

**Comment by erdkse** [comment]
> thanks @erdi - see below - could we make the left dropdown smaller so more of the filename is shown? perhaps we should call the option "Add to file" to make it even smaller?
> 
> image

I see this problem but couldn't produce a better idea to solve - implemented your suggestion - there will be always wider file names but users can read whole file name when they hover the selection - I thought that kind a solution and left sizes as it was

annnd you mentioned someone else 😄

**Pull request merged**